rabbitmq/json-exchanges.yml

```diff
@@ -0,0 +1,123 @@
+type: json
+data:
+  url: /api/exchanges
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: exchanges
+    metricNamespace: rabbitmq
+    path: $.[?(@.name=${name} && @.vhost=${vhost})]
+
+    metric:
+      - name: exchanges.message_stats.ack
+        source: message_stats/ack
+        type: long_gauge
+        label: ack messages
+        description: Number of messages delivered to clients and acknowledged
+
+      - name: exchanges.message_stats.ack.rate
+        source: message_stats/ack_details/rate
+        type: double_gauge
+        label: ack messages rate
+        unit: message/second
+        description: Number per second of messages delivered to clients and acknowledged
+
+      - name: exchanges.message_stats.confirm
+        source: message_stats/confirm
+        type: long_gauge
+        label: confirmed messages
+        description: Count of messages confirmed
+
+      - name: exchanges.message_stats.confirm.rate
+        source: message_stats/confirm_details/rate
+        type: double_gauge
+        label: confirmed messages rate
+        unit: message/second
+        description: Rate of messages confirmed per second
+
+      - name: exchanges.message_stats.deliver.get
+        source: message_stats/deliver_get
+        type: long_gauge
+        label: delivered get messages
+        description: Sum of messages delivered in acknowledgement mode to consumers, in no-acknowledgement mode to consumers, in acknowledgement mode in response to basic.get, and in no-acknowledgement mode in response to basic.get.
+
+      - name: exchanges.message_stats.deliver.get.rate
+        source: message_stats/deliver_get_details/rate
+        type: double_gauge
+        label: delivered get messages rate
+        unit: message/second
+        description: Rate per second of the sum of messages delivered in acknowledgement mode to consumers, in no-acknowledgement mode to consumers, in acknowledgement mode in response to basic.get, and in no-acknowledgement mode in response to basic.get.
+
+      - name: exchanges.message_stats.publish
```

Comment:
`exchanges.messages.published`... same thing for other metrics, will not add more comments.